CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+TBA
+ - Refactorings in Rqes-Data-Classes
+   - Remove `Csc`-Prefix from nearly all CSC data classes
```

Comment:
Improvement suggestion: Leave the old data classes, empty, but with an annotation to support automatic replacement in IDEA:

```kotlin
@Deprecated("Renamed", ReplaceWith("QtspSignatureRequest"))
data class CscSignatureRequestParameters(val foo: String)
```